File: app/src/main/java/com/itachi1706/busarrivalsg/GsonObjects/sgLTA/BusArrivalArrayObject.java
package com.itachi1706.busarrivalsg.gsonObjects.sgLTA;

/**
 * Created by Kenneth on 18/6/2015
 * for SingBuses in package com.itachi1706.busarrivalsg.GsonObjects.LTA
 */
public class BusArrivalArrayObject {
    private String ServiceNo, Operator;
    private BusArrivalArrayObjectEstimate NextBus;

    // Implemented as of 30 July 2017
    private BusArrivalArrayObjectEstimate NextBus2, NextBus3;

    private String stopCode;
    private boolean svcStatus;

    public BusArrivalArrayObjectEstimate getNextBus2() {
        return NextBus2;
    }

    public BusArrivalArrayObjectEstimate getNextBus() {
        return NextBus;
    }

    public String getOperator() {
        return Operator;
    }

    public String getServiceNo() {
        return ServiceNo;
    }

    public String getStopCode() {
        return stopCode;
    }

    public void setStopCode(String stopCode) {
        this.stopCode = stopCode;
    }

    public BusArrivalArrayObjectEstimate getNextBus3() {
        return NextBus3;
    }

    public boolean isSvcStatus() {
        return svcStatus;
    }

    public void setSvcStatus(boolean svcStatus) {
        this.svcStatus = svcStatus;
    }
}

File: app/src/main/java/com/itachi1706/busarrivalsg/GsonObjects/sgLTA/BusArrivalArrayObject.kt
@file:Suppress("PropertyName")

package com.itachi1706.busarrivalsg.gsonObjects.sgLTA

/**
 * Created by Kenneth on 18/6/2015
 * for SingBuses in package com.itachi1706.busarrivalsg.gsonObjects.sgLTA
 */
class BusArrivalArrayObject {
    val ServiceNo: String = ""
    val Operator: String = ""
    val NextBus: BusArrivalArrayObjectEstimate? = null

    // Implemented as of 30 July 2017
    val NextBus2: BusArrivalArrayObjectEstimate? = null
    val NextBus3: BusArrivalArrayObjectEstimate? = null

    var stopCode: String = ""
    var isSvcStatus: Boolean = false
}